From 1c4dc6f55836bc1b1704dafd00a79472cbc6ff65 Mon Sep 17 00:00:00 2001
From: Joe Lawrence <joe.lawrence@redhat.com>
Date: Mon, 16 Jan 2023 09:33:55 -0500
Subject: [PATCH 4/5] kpatch-build: ignore init/version-timestamp.o

Kernel v6.1+ commit 2df8220cc511 ("kbuild: build init/built-in.a just
once") split init_uts_ns and linux_banner out to
init/version-timestamp.c from init/version.c

Add init/version-timestamp.o to the list of object files that kpatch-cc
won't add to its changed_objs list.

Signed-off-by: Joe Lawrence <joe.lawrence@redhat.com>
---
 kpatch-build/kpatch-cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kpatch-build/kpatch-cc b/kpatch-build/kpatch-cc
index 2a3d264..17aae25 100755
--- a/kpatch-build/kpatch-cc
+++ b/kpatch-build/kpatch-cc
@@ -44,6 +44,7 @@ if [[ "$TOOLCHAINCMD" =~ ^(.*-)?gcc$ || "$TOOLCHAINCMD" =~ ^(.*-)?clang$ ]] ; th
 				arch/s390/kernel/vdso64/*|\
 				drivers/firmware/efi/libstub/*|\
 				init/version.o|\
+				init/version-timestamp.o|\
 				kernel/system_certificates.o|\
 				lib/*|\
 				tools/*|\
-- 
2.53.0

